roles/netbox/defaults/main.yml

```diff
@@ -0,0 +1,135 @@
+  - 'netbox_topology_views'
+  - 'netbox_inventory'
+  - 'netbox_bgp'
+
+# Quality of Life
+netbox_protection_rules:
+  "dcim.site":
+    - "status":
+        - "eq": "decommisioning"
+
+netbox_path_yaml_config: "{{ netbox_path_base }}/config"
+netbox_path_state: "{{ netbox_path_base }}/state"
+netbox_path_static: "{{ netbox_path_base }}/netbox/static"
+netbox_path_socks: "{{ netbox_path_base }}/socks"
+netbox_path_socks_uwsgi: "{{ netbox_path_socks }}/uwsgi.sock"
+netbox_path_nginx_config: "{{ netbox_path_base }}/nginx.conf"
+netbox_path_uwsgi_config: "{{ netbox_path_base }}/uwsgi.ini"
+netbox_file_config_py: "{{ netbox_path_base }}/netbox/netbox/configuration.py"
+
+netbox_container_port: 8000
+
+netbox_container_name: "netbox"
+netbox_container_env: {}
+netbox_container_ports: []
+netbox_container_labels_merged: >-
+  {{ netbox_container_labels_base |
+  combine(netbox_container_labels | default({})) }}
+netbox_container_labels_base:
+  version: "{{ netbox_version }}"
+netbox_container_volumes:
+  - "{{ netbox_path_base ~ '/configuration.py' ~ ':' ~ netbox_file_config_py }}"
+  - "{{ netbox_path_yaml_config ~ ':' ~ netbox_path_yaml_config }}"
+  - "{{ netbox_path_state ~ ':' ~ netbox_path_state }}"
+  - "{{ netbox_postgres_socket ~ ':' ~ netbox_postgres_socket }}"
+  - "{{ netbox_path_static ~ ':' ~ netbox_path_static }}"
+  - "{{ netbox_path_socks ~ ':' ~ netbox_path_socks }}"
+  - "{{ netbox_path_nginx_config }}:/etc/nginx/conf.d/netbox.conf"
+  - "{{ netbox_path_uwsgi_config ~ ':' ~ netbox_path_uwsgi_config }}"
+  - "/etc/passwd:/etc/passwd:ro"
+
+netbox_container_restart_policy: "always"
+netbox_container_image_state: "present"
+netbox_container_image_force_pull: true
+netbox_container_image_reference: >-
+  {{
+    netbox_container_image_repository
+    ~ ":"
+    ~ netbox_container_image_tag | default('v' ~ netbox_version ~ '-r0')
+  }}
+
+netbox_container_nginx_image_reference: >-
+  {{
+    netbox_container_nginx_image_repository
+    ~ ":"
+    ~ netbox_container_nginx_image_tag | default('bookworm')
+  }}
+
+netbox_container_image_repository: >-
+  {{
+    (
+    container_registries[netbox_container_image_registry]
+    | default(netbox_container_image_registry)
+    )
+    + '/'
+    + netbox_container_image_namespace | default('')
+    + netbox_container_image_name
+  }}
+
+netbox_container_nginx_image_repository: >-
+  {{
+    (
+    container_registries[netbox_container_nginx_image_registry]
+    | default(netbox_container_nginx_image_registry)
+    )
+    + '/'
+    + netbox_container_nginx_image_namespace | default('')
+    + netbox_container_nginx_image_name
+  }}
+
+netbox_container_image_registry: "docker-oss.nexus.famedly.de"
+netbox_container_image_name: "netbox"
+
+netbox_container_nginx_image_registry: "docker.io"
+netbox_container_nginx_image_namespace: "nginxinc/"
+netbox_container_nginx_image_name: "nginx-unprivileged"
+
+netbox_postgres_name: "netbox"
+netbox_postgres_user: "netbox"
+netbox_postgres_password: ~
```

Comment:
This should probably be undefined instead of set to `None`, so that accessing the variable fails completely